tests/crd_test_util.go

```diff
@@ -122,3 +156,463 @@ func checkPodsDisasterTolerance(allPods []corev1.Pod) error {
 	}
 	return nil
 }
+
+func (ctu *CrdTestUtil) DeleteTidbClusterOrDie(tc *v1alpha1.TidbCluster) {
+	err := ctu.cli.PingcapV1alpha1().TidbClusters(tc.Namespace).Delete(tc.Name, &metav1.DeleteOptions{})
+	if err != nil {
+		if errors.IsNotFound(err) {
+			return
+		}
+		slack.NotifyAndPanic(err)
+	}
+}
+
+func (ctu *CrdTestUtil) WaitTidbClusterReadyOrDie(tc *v1alpha1.TidbCluster, timeout time.Duration) {
```

Comment:
why not use the similary function `operatorActions.WaitForTidbClusterReady` which takes a TidbCluster CR too?

Reply:
I plan we will finally discard the function which operate the tidbcluster in `operatorActions`

Reply:
ok